ee/tabby-webserver/src/service/background_job/db.rs

```diff
@@ -37,4 +38,158 @@ impl DbMaintainanceJob {
             .await?;
         Ok(())
     }
+
+    pub async fn retention(now: DateTime<Utc>, db: DbConn) -> tabby_schema::Result<()> {
```

Comment:
```suggestion
    async fn data_retention(now: DateTime<Utc>, db: DbConn) -> tabby_schema::Result<()> {
```

just call it in `cron`